lib/trino-aws/src/main/java/io/trino/aws/AwsCredentialsProviderConfig.java

```diff
@@ -139,6 +139,7 @@ public Optional<String> getAwsRegionName()
         return awsRegionName;
     }
 
+    @LegacyConfig("region-name")
```

Comment:
Is it OK to add something like this even though it's not required for anything in Trino itself?